docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03 - Sites.md

```diff
@@ -0,0 +1,40 @@
+# Sites
+
+## Overview Study
+
+**Sites** or **centres** are terms used interchangeably in LORIS.
+**Sites** are defined in the `psc` table of the database.
+
+LORIS' default schema defines the first psc (CenterID=1) as the
+Data Coordinating Center or "DCC".  This site is generally used as
+dummy data and is assumed by the codebase to store non-study data,
```

Comment:
would finish the sentence after "...non-study data**.**" and then start a new sentence "**Therefore,** it **is** not recommended..."